File: src/tasks/cli/feature/remove/RemoveArguments.java
package tasks.cli.feature.remove;

import omnia.data.structure.List;
import tasks.cli.command.common.simple.SimpleArguments;
import tasks.model.ObservableTask;

/** Model for parsed Remove command arguments. */
public final class RemoveArguments extends SimpleArguments {
  RemoveArguments(List<ObservableTask> tasks) {
    super(tasks);
  }

  /** The tasks to remove from the store. */
  @Override
  public List<ObservableTask> tasks() {
    return super.tasks();
  }
}

File: src/tasks/cli/feature/remove/RemoveArguments.kt
package tasks.cli.feature.remove

import omnia.data.structure.List
import tasks.cli.command.common.simple.SimpleArguments
import tasks.model.Task

/** Model for parsed Remove command arguments.  */
class RemoveArguments internal constructor(tasks: List<Task>, private val force: Boolean) : SimpleArguments(tasks) {
  /** Force. auto-confirm deletions, bypassing manual confirmations.  */
  fun force(): Boolean {
    return force
  }
}